class RepoError(Exception):
    pass
class BusinessError(Exception):
    pass